servers/tests/org.fusesource.ide.server.tests/src/main/java/org/fusesource/ide/server/tests/util/FuseESBMockRuntimeCreationUtil.java

```diff
@@ -52,13 +67,42 @@ public static boolean create6xRuntimeMock(String runtimeId, IPath runtimePath) {
 		return runtimeCreated;
 	}
 	
+	/**
+	 * creates a mock runtime folder structure 
+	 * 
+	 * @param runtimeId		the runtime type id to use
+	 * @param runtimePath	the path where to create the structure
+	 * @return	true on success
+	 */
+	public static boolean create7xRuntimeMock(String runtimeId, IPath runtimePath) {
+		boolean runtimeCreated = true;
+		
+		if (isSupported7xRuntime(runtimeId)) {
+			createBaseFuseFolderSkeleton(runtimePath);
+			try {
+				copyFuseJarToSkeleton(runtimeId, runtimePath);
+			} catch (CoreException ex) {
+				runtimeCreated = false;
+			}
+		}		
+		
+		return runtimeCreated;
+	}
+	
 	private static boolean isSupported6xRuntime (String runtimeId) {
 		for (String id : SUPPORTED_6X_RUNTIMES) {
```

Comment:
if using List<String>, it can be written simpler:
return SUPPORTED_6X_RUNTIMES.contains(runtimeId);

Reply:
up